src/main/java/ejercicio/banco/MainProcess.java

```diff
@@ -16,55 +17,82 @@ public static void main(String[] args) throws FileNotFoundException {
 
-        // Creates the bank
-        Bank bestBank = bankManager.manage(bankFile);
+        BankService bankService;
+        AccountService accountService;
+        PaymentService paymentService;
 
-        // Creates account list with Account manager
-        List<Account> accounts = accountManager.manage(accountFile);
+        Scanner scann = new Scanner(System.in);
+        System.out.println("Do you wish to work with csv or sql?(type either csv or sql)");
 
-        // Stores all the accounts into our bank before payments
-        bestBank.setUsers(accounts);
+        //Waits for input
+        String type = scann.nextLine();
 
+        //If using sql
+        if (type.equals("sql")) {
 
-        //Writes accounts before payments
-        paymentManager.fwriter(String.join(File.separator, "src", "main", "resources", "csv", "accountsBeforePayments.csv"), bestBank);
+            //Because sql repositories are not implemented yet, it will just terminate
+            System.out.println("You choosed sql, you choosed wrong");
 
+            bankService = new BankServiceImpl(new MySqlBankRepository());
+            accountService = new AccountServiceImpl(new MySqlAccountRepository());
+            paymentService = new PaymentServiceImpl(new MySqlPaymentRepository());
+        }
+
+        //If using csv
+        else {
+
+            bankService = new BankServiceImpl(new CsvBankRepository());
+            accountService = new AccountServiceImpl(new CsvAccountRepository());
+            paymentService = new PaymentServiceImpl(new CsvPaymentRepository());
+
+
+            // File names might change depending on file location
+            String bankFile = String.join(File.separator, "src", "main", "resources", "csv", "bank.csv");
+            String accountFile = String.join(File.separator, "src", "main", "resources", "csv", "accounts.csv");
+            String paymentFile = String.join(File.separator, "src", "main", "resources", "csv", "payments.csv");
+
+            // Creates the bank
+            Bank bestBank = bankService.manage(bankFile);
+
+            // Creates account list with Account manager
+            List<Account> accounts = accountService.manage(accountFile);
 
-        //This is the code that will be repeated if user wants
-        while (yesNo) {
-            List<Payment> payments = paymentManager.manage(paymentFile, bestBank);
+            // Stores all the accounts into our bank before payments
+            bestBank.setUsers(accounts);
 
 
-            Scanner scan = new Scanner(System.in);
+            //Writes accounts before payments
+            paymentService.fwriter(String.join(File.separator, "src", "main", "resources", "csv", "accountsBeforePayments.csv"), bestBank);
 
 
-            Boolean repeat = true;
-            while (repeat) {
+            //This is the code that will be repeated if user wants
+            while (yesNo) {
```

Comment:
This `yesNo` variable doesn't tell its real purpose. I would rename it to something more descriptive like `repeat` or something similar.